docs/get-started/alerting.md

```diff
@@ -129,48 +227,65 @@ templates:
 6. Click **Save and Exit** to close the page and go to the **Alert Rules** tab where you can review, edit and silence your new alert.
 
 ## Silence alerts
-Create a silence when you want to stop notifications from one or more alerting rules.
+Create a silence when you want to stop notifications from one or more alerting rules for a very specific amount of time. Silences default to today’s current date and  have a default duration of two hours.
 
-Silences stop notifications from being sent to your specified contact points.
+During a silence, PMM continues to track metrics but does not trigger alerts or send notifications to any specified contact points. Once the silence expires alerts and notifications will resume.
+
+For example, you can create a silence to suppress a notifications for a specific period of time.
 
 Silenced alerts are still recorded under **Alerting > Fired Alerts** so that you can review them later. Silenced alerts are disabled for as long as it's specified in the Silence Duration or until you remove a silence.
 
-For information on creating silences, see [About alerting silences](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/create-silence/) in the Grafana documentation.
+#### Add a silences
+
+You can easily silence an alert from the **Fired alerts** page or from the **Alert rules** page.
+You can also create a silence from the **Silences** page, but here you would also need to define labels that match the alert that you want to silence.
+
+To create a new silence from the **Silences** page:
+
+1. Click the **New Silence** button.
+2. Select the start and end date to indicate when the silence should go into effect and expire.
+3. Optionally, update the duration to alter the time for the end of silence in the previous step to correspond to the start plus the duration.
+4. Enter one or more matching labels by filling out the **Name** and **Value** fields. Matchers determine which rules the silence will apply to.
+5. Enter a comment.
+6. Review the affected alert instances that will be silenced.
+7. Click **Sumbit** to create the silence.
+
+For more information on working with silences, see [About alerting silences](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/create-silence/) in the Grafana documentation.
 
 ## Deprecated alerting options
- PMM 2.31 introduced Percona Alerting which replaces the old Integrated Alerting in previous PMM versions. In addition to full feature parity, Percona Alerting includes additional benefits like Grafana-based alert rules and a unified, easy-to-use alerting command center on the **Alerting** page.
 
-### Alerting compatibility 
+PMM 2.31 introduced Percona Alerting which replaces the old Integrated Alerting in previous PMM versions. In addition to full feature parity, Percona Alerting includes additional benefits like Grafana-based alert rules and a unified, easy-to-use alerting command center on the **Alerting** page.
```

Comment:
This first paragraph should be at the very beginning of the Percona Alerting Intro